### PR TITLE
Update che4z-installing.adoc

### PR DESCRIPTION
**Pull request by DStatWriter**
Signed-off-by: Daniel Statham <daniel.statham@broadcom.com>

### What does this PR do?
Expands on the install section, removing the need for users to click through to another page

### What issues does this PR fix or reference?
#14967 #14981

### CODE REVIEW

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 1]
```diff
@@ -13,4 +13,50 @@ summary:
 
 :context: installing-che4z
 
-To download and install the Eclipse Che4z plugins, follow the instructions on link:https://projects.eclipse.org/projects/ecd.che.che4z/downloads[this page].
\ No newline at end of file
+=== Prerequisites
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
.Prerequisites
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 2]
```diff
@@ -13,4 +13,50 @@ summary:
...
+- Minimum of 2 CPUs and 4GB RAM
+- Docker
+- Internet access
+
+'''
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 3]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+=== Installation
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
.Procedure
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 4]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+===== *Follow these steps:*
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 5]
```diff
@@ -13,4 +13,50 @@ summary:
...
+. Install Eclipse Che by running the following command, and replacing <DATA_FOLDER> with the location of user data on the host machine:
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
. Install Eclipse Che by running the following command. Replace `__<DATA_FOLDER>__` with the location of user data on the host machine:
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 6]
```diff
@@ -13,4 +13,50 @@ summary:
...
+----
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc:27]:
```suggestion
+
[subs="+quotes"]
----
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 7]
```diff
@@ -13,4 +13,50 @@ summary:
...
+	docker run -it --rm -e CHE_MULTIUSER=true -e CHE_HOST=`hostname -f` -e
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
$ docker run -it --rm -e CHE_MULTIUSER=true -e CHE_HOST=`hostname -f` -e \
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 8]
```diff
@@ -13,4 +13,50 @@ summary:
...
+	CHE_WORKSPACE_VOLUME=/var/run/docker.sock:/var/run/docker.sock -v
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
  CHE_WORKSPACE_VOLUME=/var/run/docker.sock:/var/run/docker.sock -v \
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 9]
```diff
@@ -13,4 +13,50 @@ summary:
...
+	/var/run/docker.sock:/var/run/docker.sock -v /<DATA_FOLDER>:/data eclipse/che:7.0.0-beta-4.0 start
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
  /var/run/docker.sock:/var/run/docker.sock -v /__<DATA_FOLDER>__:/data eclipse/che:7.0.0-beta-4.0 start
```
Is it really dependent on Che v7.0.0-beta-4.0? If not, suggest using a placeholder for the latest version.

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 10]
```diff
@@ -13,4 +13,50 @@ summary:
...
+----
+- For more information on configuring Eclipse Che with Docker, see https://www.eclipse.org/che/docs/che-6/docker-multi-user.html[here]
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
+
For more information on configuring Eclipse Che with Docker, see https://www.eclipse.org/che/docs/che-6/docker-multi-user.html[Installation on Docker].
```
Perhaps we should be pointing to Che 7 docs and local installation methods?

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 11]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+. Log in to Che using the default administrator account:
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
. Log in to Che using the default administrator account (username = `admin`, password = `admin`).
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 12]
```diff
@@ -13,4 +13,50 @@ summary:
...
+  - (username = admin, password = admin).
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 13]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+. (Optional) Configure the Keycloak security server by following the instructions https://www.eclipse.org/che/docs/che-6/user-management.html#che-and-keycloak[here].
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
. (Optional) Configure the Keycloak security server by following the instructions https://www.eclipse.org/che/docs/che-6/user-management.html#che-and-keycloak[Managing users].
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 14]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+. Download and unzip the installer.
+
+. Run *start_installer.sh* (on Linux) or *start_installer.cmd* (on Windows).
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
. Run `start_installer.sh` (on Linux) or `start_installer.cmd` (on Windows).
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 15]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+. When prompted, enter the Che URL and administrator account credentials.
+
+You installed Eclipse Che and the Che4z extensions.
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 16]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+'''
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 17]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+=== Creating a Workspace
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
== Creating a workspace
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 18]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+===== *Follow these steps:*
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 19]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+1. Log in to Che. Use your credentials, or the default administrator account credentials:
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
. Log in to Che. Use your credentials or the default administrator account (username = `admin`, password = `admin`).
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 20]
```diff
@@ -13,4 +13,50 @@ summary:
...
+  - (username = admin, password = admin).
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 21]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+2. To create a new workspace, select *Workspaces - Add Workspace*.
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
. Select *Workspaces -> Add Workspace*.
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 22]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+3. In *Select Stack*, select *view all*, then choose *Broadcom Mainframe* stack.
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
. In *Select Stack*, select *view all*, then choose the *Broadcom Mainframe* stack.
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 23]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+4. Create and open the workspace.
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
. Create and open the workspace.
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 24]
```diff
@@ -13,4 +13,50 @@ summary:
...
+
+You created a workspace and can now use the Eclipse Che4z extensions.
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
A workspace has been created that can use the Eclipse Che4z extensions.
```

**Review by rkratky — CHANGES_REQUESTED** [review]

**Review by rkratky — CHANGES_REQUESTED** [review]
Thanks, @DStatWriter. I added some suggestions.

**Comment by themr0c** [comment]
this still needs to have the commits signed off. Please, rebase, squash the commits, and add your sign-off line to the commit message.

**Comment by DStatWriter** [comment]
> this still needs to have the commits signed off. Please, rebase, squash the commits, and add your sign-off line to the commit message.



> this still needs to have the commits signed off. Please, rebase, squash the commits, and add your sign-off line to the commit message.

Hey Fabrice, I did with the original submit, but it has to be added to each individual commit created by accepting change suggestions? I guess im best just re-submitting the changed ersion fresh with a single sign off, which i think is what you might have been suggesting here.

Dan

**Comment by themr0c** [comment]
@DStatWriter Yes, that's exactly what I meant!

**Comment by DStatWriter** [comment]
> @DStatWriter Yes, that's exactly what I meant!

Ok great, thanks

Dan

**Comment by themr0c** [comment]
@DStatWriter and don't forget to look also at the suggestions that are in the "14 hidden conversations".

**Comment by themr0c** [comment]
(and then again the same amend signoff squash and push force of course)

**Comment by DStatWriter** [comment]
Re-validated PR with corrected email

**Pull request closed without merging**